src/dmd/root/outbuffer.h

```diff
@@ -46,7 +46,7 @@ struct OutBuffer
     void reserve(size_t nbytes);
     void setsize(size_t size);
     void reset();
-    //void write(const void *data, d_size_t nbytes);
+    void write(const void *data, size_t nbytes);
```

Comment:
This should be d_size_t due to OSX.

Reply:
All other methods here use `size_t`... only DMD 2.079-2.081 IIRC are affected by the temporary size_t interop regression; LDC has never suffered from it.

Reply:
It's not there only for the size_t regression, there are some platforms where you will suffer from it. ;-)

Reply:
Regardless, I'll add a test for this in the cxx-unittester, and I'll be forced to change it to d_size_t anyway...

Reply:
> there are some platforms where you will suffer from it

32-bit macOS isn't supported any longer, and all others use `d_size_t = size_t` as of now.

> I'll be forced to change it to d_size_t

Alright then. I still count about 45 `size_t` occurrences in the C++ headers, and most of those are function params.